doc/api/zlib.markdown

```diff
@@ -360,7 +360,7 @@ Decompress a raw Buffer with Unzip.
 [content-encoding]: http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.11
 [Memory Usage Tuning]: #zlib_memory_usage_tuning
 [zlib documentation]: http://zlib.net/manual.html#Constants
-[options]: #zlib_options
+[options]: #zlib_class_options
```

Comment:
this is strange. because the source does not have _class_ but the link does have it after building docs (without this patch).

I'm going to run the docs again with these ch